exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -252,6 +251,44 @@ impl JavaServiceRuntime {
     }
 }
 
+/// Wrapper for InitArgsBuilder.
+struct JvmArgsBuilder {
+    internal: InitArgsBuilder,
+    jvm_args_line: String,
+}
+
+impl JvmArgsBuilder {
+    /// Create a new JvmArgsBuilder
+    pub(self) fn new() -> Self {
```

Comment:
Just wondering: why is it `pub(self)`, not `pub` (as struct is private anyway)?

Reply:
It's not used outside module so it's equal to private. Which is ok in case we're not going to put it outside.